iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/manager/pipe/metric/PipeConfigNodeReceiverMetrics.java

```diff
@@ -0,0 +1,150 @@
+  private Timer transferConfigPlanTimer = IoTDBMetricManager.getInstance().createTimer();
+  private Timer transferConfigSnapshotPieceTimer = IoTDBMetricManager.getInstance().createTimer();
+  private Timer transferConfigSnapshotSealTimer = IoTDBMetricManager.getInstance().createTimer();
+
+  private static final String RECEIVER = "pipeConfigNodeReceiver";
+
+  private PipeConfigNodeReceiverMetrics() {}
+
+  public void recordHandShakeConfigNodeV1Timer(long costTimeInNanos) {
+    handShakeConfigNodeV1Timer.updateNanos(costTimeInNanos);
+  }
+
+  public void recordHandShakeConfigNodeV2Timer(long costTimeInNanos) {
+    handShakeConfigNodeV2Timer.updateNanos(costTimeInNanos);
+  }
+
+  public void recordTransferConfigPlanTimer(long costTimeInNanos) {
+    transferConfigPlanTimer.updateNanos(costTimeInNanos);
+  }
+
+  public void recordTransferConfigSnapshotPieceTimer(long costTimeInNanos) {
+    transferConfigSnapshotPieceTimer.updateNanos(costTimeInNanos);
+  }
+
+  public void recordTransferConfigSnapshotSealTimer(long costTimeInNanos) {
+    transferConfigSnapshotSealTimer.updateNanos(costTimeInNanos);
+  }
+
+  @Override
+  public void bindTo(AbstractMetricService metricService) {
+    bindToTimer(metricService);
+  }
+
+  private void bindToTimer(AbstractMetricService metricService) {
+    handShakeConfigNodeV1Timer =
+        metricService.getOrCreateTimer(
+            Metric.PIPE_CONFIGNODE_RECEIVER.toString(),
+            MetricLevel.IMPORTANT,
+            Tag.NAME.toString(),
+            RECEIVER,
+            Tag.TYPE.toString(),
+            "handShakeConfigNodeV1");
+
+    handShakeConfigNodeV2Timer =
+        metricService.getOrCreateTimer(
+            Metric.PIPE_CONFIGNODE_RECEIVER.toString(),
+            MetricLevel.IMPORTANT,
+            Tag.NAME.toString(),
+            RECEIVER,
+            Tag.TYPE.toString(),
+            "handShakeConfigNodeV2");
+
+    transferConfigPlanTimer =
+        metricService.getOrCreateTimer(
+            Metric.PIPE_CONFIGNODE_RECEIVER.toString(),
+            MetricLevel.IMPORTANT,
+            Tag.NAME.toString(),
+            RECEIVER,
+            Tag.TYPE.toString(),
+            "transferConfigPlan");
+
+    transferConfigSnapshotPieceTimer =
+        metricService.getOrCreateTimer(
+            Metric.PIPE_CONFIGNODE_RECEIVER.toString(),
+            MetricLevel.IMPORTANT,
+            Tag.NAME.toString(),
+            RECEIVER,
+            Tag.TYPE.toString(),
+            "transferConfigSnapshotPiece");
+
+    transferConfigSnapshotSealTimer =
+        metricService.getOrCreateTimer(
+            Metric.PIPE_CONFIGNODE_RECEIVER.toString(),
+            MetricLevel.IMPORTANT,
+            Tag.NAME.toString(),
+            RECEIVER,
+            Tag.TYPE.toString(),
+            "transferConfigSnapshotSeal");
+  }
+
+  @Override
+  public void unbindFrom(AbstractMetricService metricService) {
+    unBindToTimer(metricService);
```

Comment:
If you use this style, it's unbindFromTimer.. BTW, personally I deem it better to use "unbind" to be consistent with the dataNode and other module's style. Besides, unbind is a single word too.